webapp/src/components/post_type_mstmeetings/post_type_mstmeetings.tsx

```diff
@@ -108,10 +107,9 @@ export default class PostTypeMSTMeetings extends React.PureComponent<Props> {
                             target='_blank'
                             href={props.meeting_link}
                         >
-                            <i
-                                style={style.buttonIcon}
-                                dangerouslySetInnerHTML={{__html: Svgs.VIDEO_CAMERA_3}}
-                            />
+                            <i style={style.buttonIcon}>
+                                <Icon/>
```

Comment:
Was the `Icon` component used elsewhere before being introduced here?

Reply:
Yes, it was used when registering the channel header icon.